bin/git-forgit

```diff
@@ -1002,6 +1003,98 @@ _forgit_ignore_clean() {
     [[ -d "$FORGIT_GI_REPO_LOCAL" ]] && rm -rf "$FORGIT_GI_REPO_LOCAL"
 }
 
+_forgit_worktree_preview() {
+    local sha
+    # trailing space in  grep to avoid matching worktrees with a common path
+    sha=$(git worktree list | grep "$1 " | awk '{print $2}')
+    # bare git-dir has no history
+    [[ "$sha" == "(bare)" ]] && return
+    _forgit_worktree_preview_git_opts=()
+    _forgit_parse_array _forgit_worktree_preview_git_opts "$FORGIT_WORKTREE_PREVIEW_GIT_OPTS"
+    # the trailing '--' ensures that this works for branches that have a name
+    # that is identical to a file
+    git log "$sha" "${_forgit_worktree_preview_git_opts[@]}" --
+}
+
+_forgit_worktree_jump() {
+    _forgit_inside_work_tree || _forgit_inside_git_dir || return 1
+    local count tree opts
+    count=$(git worktree list | wc -l)
+
+    [[ $count -eq 1 ]] && return 1
+
+    opts="
+        $FORGIT_FZF_DEFAULT_OPTS
+        +s +m --tiebreak=index
+        --preview=\"$FORGIT worktree_preview {1}\"
+        $FORGIT_WORKTREE_JUMP_FZF_OPTS
+    "
+
+    tree=$(git worktree list | awk '{print $1}' | FZF_DEFAULT_OPTS="$opts" fzf)
+    [[ -z "$tree" ]] && return 1
+    echo "$tree"
```

Comment:
- Using `cd` directly in the function only changes the current directory in the script child process, not in the terminal process where the script is called
- Any preference between `cd` and `pushd`?

Reply:
This is tricky, because the actual change of the directory is currently implemented in the shell plugin. Some users (e.g. me) do not use forgit with the plugin but in the form of a subcommand of `git`. Calling `git forgit worktree_jump` won't work at the moment, though. IMO we should find a way to make this work correctly in both usecases, or otherwise rename the function (because right now it actually does not jump).

Reply:
We could rename the function to `worktree_select` - `gws`
(but also offer the `gwj` alias which combines `cd` and `worktree_select`)

Reply:
Given that there is no way to manipulate the directory of the parent process, I think this is the best compromise we can achieve. @carlfriedrich WDYT?